crm_create_project/wizards/crm_create_project.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2024 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+from odoo import _, fields, models
+
+
+class CrmCreateProject(models.TransientModel):
+    _name = "crm.create.project"
+    _description = "Wizart to create Project from Lead/Opportunity"
+
+    project_name = fields.Char()
+    project_description = fields.Html()
+    lead_id = fields.Many2one("crm.lead")
+
+    def create_project(self):
+        project = (
+            self.env["project.project"]
+            .sudo()
```

Comment:
thought: This sudo puts me to think... The salesman should have permissions to create projects? How useful is this module if you can create a project to plan it, but can't edit it later? Maybe this model should be only available to users with CRM *and* project permissions?

Reply:
This module is designed to enable the salesperson to create projects from lead/opportunity. 
Generally, the sales person will not edit the project created from the opportunity. The project will be used by a person with project permissions, occasionally the sales person may be that user if they have project permissions, but usually it will be another user such as a project manager.

Reply:
OK that makes sense. But in that case, please make it clearer in the module. It should be explained in the readme. Maybe a warning message in the wizard would help too (something like: "you won't be able to edit this project after creating it", displayed for non-project-manager salespeople).